brave-spancollector-kafka/src/main/java/com/github/kristofa/brave/kafka/KafkaSpanCollector.java

```diff
@@ -91,6 +100,7 @@ public void close() {
         }
         executorService.shutdown();
         producer.close();
+        metricsHandler.incrementDroppedSpans(queue.size());
```

Comment:
edge case, but if we don't call `awaitTermination` or `isTerminated` after `shutdown`, would this value be accurate?